File: solutions/215. Kth Largest Element in an Array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>(Collections.reverseOrder());
        for (int x : nums) pq.add(x);
        while(k-1 > 0) {pq.remove(); k--;}
        return pq.peek();
    }
}

File: ..solutions/215. Kth Largest Element in an Array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int, vector<int>> q(nums.begin(), nums.end());
        while(--k) {
            q.pop();
        }
        return q.top();
    }
};
